src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -102,7 +102,6 @@ static GrantStmt * GenerateGrantStmtForRights(ObjectType objectType,
 											  bool withGrantOption);
 static List * GetObjectsForGrantStmt(ObjectType objectType, Oid objectId);
 static AccessPriv * GetAccessPrivObjectForGrantStmt(char *permission);
-static RoleSpec * GetRoleSpecObjectForGrantStmt(Oid roleOid);
```

Comment:
funtimes, enterprise has these definitions reordered, hence the merge conflict :/
couldn't find a way to change this without conflict. Only by keeping this function and using a copy for my purpose, but that seems wrong. Will fix conflicts after approval.